lib/bgs/service.rb

```diff
@@ -273,11 +273,9 @@ def create_note(claim_id, note_text)
     private
 
     def service
-      external_key = @user.common_name || @user.email
-
       @service ||= BGS::Services.new(
         external_uid: @user.icn || @user.uuid,
-        external_key: external_key
+        external_key: @user.email
```

Comment:
Can you also make this change in `bgs/base_service.rb`? They have a similar way to set `external_key`, but we should prob make these consistent